acme/acme/client.py

```diff
@@ -255,16 +258,23 @@ def retry_after(cls, response, default):
         :rtype: `datetime.datetime`
 
         """
-        retry_after = response.headers.get('Retry-After', str(default))
+        retry_after = response.headers.get('Retry-After', str(default)).strip()
         try:
             seconds = int(retry_after)
         except ValueError:
             # pylint: disable=no-member
-            decoded = werkzeug.parse_date(retry_after)  # RFC1123
-            if decoded is None:
+            when = parsedate_tz(retry_after)
```

Comment:
In any case, `parsedate_tz` handles [each of the 2616 cases](https://www.w3.org/Protocols/rfc2616/rfc2616-sec3.html#sec3.3.1).  However I'm going to remove any special casing in our code for the RFC 850 subcase.  No ACME servers will ever speak HTTP/1.0 or issue RFC 850 dates, so we don't need extra code to sure up python 2.6's implementation of that deprecated format.